src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -145,6 +145,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
                   className={venueChatTabStyles}
                   onClick={selectVenueChat}
                 >
+                  <FontAwesomeIcon icon={faCommentDots} size="sm" />
```

Comment:
`ButtonNG` supports icons with more concise syntax. Example
 ```tsx
        <ButtonNG
          isLink
          className="RunTabView__advanced"
          linkTo={adminNGSettingsUrl(venueId)}
          iconName={faCog}
        >
          Advanced Settings
        </ButtonNG>
```

Reply:
fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54